### ISSUE TITLE
Problem for creating classification map

### ISSUE THREAD

**Issue by RanRan-Margaret** [ZhengPeng7/W-Net-Keras#4]
From original paper"we used the same method as density generation but 
used a larger window size and spread parameter. Once the 
blurred map is generated, we use binary thresholding to 
create classification map to train the reinforcement branch. 
In our experiments, we used a threshold (th) = 0.001 ",I can't find the  Corresponding content in your code ,will you help me? Please tell me the  specific location in your code.

**Comment by ZhengPeng7** [comment]
Thanks for your interest:)
I'm sorry that I didn't implement this thresholding on attention in my project since I think the threshold is a hyper-parameter, which may be not good at generalization in the network and also turn the soft attention to a hard mask.
